toolkit/tools/pkg/imagecustomizerlib/partitionutils.go

```diff
@@ -121,7 +121,7 @@ func findRootfsPartition(diskPartitions []diskutils.PartitionInfo, buildDir stri
 
 		// Skip over file-system types that can't be used for the rootfs partition.
 		switch diskPartition.FileSystemType {
-		case "ext2", "ext3", "ext4", "xfs":
+		case "ext2", "ext3", "ext4", "xfs", "btrfs":
```

Comment:
It might be worth adding a comment stating that subvolumes aren't yet supported.